### DISCUSSION TITLE
请按承诺退款

### DISCUSSION THREAD

**Discussion by David68326** [getlantern/lantern-forum-cn#536] [General]
@Mike777Khoo 
   4月24号答应我14天左右可以办理退款，但现在1个多月过去了还没办理，请尽快安排，谢谢！
请参看原始帖子是 #465 
   作为蓝灯好多年的老会员，从去年10月份就开始忍受蓝灯的无法正常使用，之所以2月份又续费2年，就是一直期待你们能解决问题，并且还为你们提供发现的问题，但现在过去8、9个月了，你们却依然无法解决问题，不得已我已经换用了其它的VPN，否则连这个论坛都上不来。希望你们正视自己的问题，在退款问题上不要再敷衍，让老用户彻底失望。

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/fd1f7c63-21ab-4e34-a58a-20f408082a61)
@David68326 您好. 技术人员帮您试过好几次退款了. 但是由于帐号购买太久了, 已经无法退款.
他们说您可以试试看从支付系统或银行退款吗? 我们的系统已经无法允许退款了. 一般退款都是在两个月里面就必须要求退款的.

**Reply by David68326** [reply]
你这个说法很奇怪啊，钱你们收走了，让我找支付系统或银行退款？系统不能退，你们可以自己操作退给我不是吗？

**Comment by David68326** [comment]
我是2月份买的，4月份提出退款的，到现在不就是2.3个月吗？钱你们已经收了，这么找银行退？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月1日(周六) 凌晨0:31
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
image.png (view on web)
 @David68326 您好. 技术人员帮您试过好几次退款了. 但是由于帐号购买太久了, 已经无法退款.
 他们说您可以试试看从支付系统或银行退款吗? 我们的系统已经无法允许退款了. 一般退款都是在两三个月里面就必须要求退款的.

**Comment by David68326** [comment]
你们收了钱，用正常的支付方式返还给我不就可以了吗？你们没有付过款吗？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月1日(周六) 凌晨0:31
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
image.png (view on web)
 @David68326 您好. 技术人员帮您试过好几次退款了. 但是由于帐号购买太久了, 已经无法退款.
 他们说您可以试试看从支付系统或银行退款吗? 我们的系统已经无法允许退款了. 一般退款都是在两三个月里面就必须要求退款的.

**Reply by zwzld** [reply]
软件非常不友好，一下买了2年，都不知道怎么退款

**Comment by David68326** [comment]
你们不能正常提供服务，还不想退款，有道理吗？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月1日(周六) 凌晨0:31
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
image.png (view on web)
 @David68326 您好. 技术人员帮您试过好几次退款了. 但是由于帐号购买太久了, 已经无法退款.
 他们说您可以试试看从支付系统或银行退款吗? 我们的系统已经无法允许退款了. 一般退款都是在两三个月里面就必须要求退款的.

**Comment by MaxMKMax** [comment]
@David68326 您好! 不是不想退款. 是支付系统里无法操作了. 它有固定的时间. 

而且专业版我们是有让用户先试用免费的, 才购买.

您放心, 我再帮您问看有什么方法吗. 支付部门是说您可以试看从银行取消购买 / 退款看看.
我们使用第三方支付系统的, 能不能在系统退款我们决定不了. 希望您理解.

**Reply by David68326** [reply]
不管是不是第三方系统，钱你们已经收了。退款可以直接把钱退到我银行卡或者支付宝而无需通过第三方系统，能支付宝收款不能支付宝退款？这么简单的事该怎么处理你们做公司的会不懂吗？
至于购买前有没有试用已经没有关系了，购买之后你们的服务就没有能够正常提供过，就像人家让你试吃个Icecream，你吃了之后觉得味道不错，买了一堆回家发现都是变味的，这种情况我要求退款这有什么问题吗？作为好多年的专业付费版老用户，如果可以正常用，我会没事找事来退款吗？记住，退款你们没有能力解决正常使用的问题才造成的。
如果不退款的话，我会一直来的。

**Reply by David68326** [reply]
我刚刚咨询了当时付款的第三方支付宝，1年以内的交易都可以退款，但必须是你们提出申请他们才能处理，你们申请后支付宝第二天就能退款给我。请看附件截图。

![支付宝退款](https://github.com/getlantern/lantern-forum-cn/assets/157975130/c1ff2dab-c4ac-4794-949d-1abad88250c5)

**Comment by MaxMKMax** [comment]
@David68326 您好. 我们没有说不退款. 您给我们几天工作日时间, 我们尽力帮您争取的. 周末团队不在线, 等他们上线我来帮您问好不好? 请给我一周的时间我来给您答复. 谢谢您的耐心等待!

**Comment by MaxMKMax** [comment]
你一周后如果没消息再来问我. 我这边有消息了会第一时间通知您.

**Reply by David68326** [reply]
好的，谢谢！

**Reply by MaxMKMax** [reply]
@David68326 您不客气. 尊重客户的体验感是我们的首要原则. 我们一定会善待我们的用户的. 谢谢您对蓝灯的支持!

**Comment by David68326** [comment]
您好，我依然没有收到退款，请尽快办理。


上个邮件我已经发支付平台的回复给您了，一年内的付款都可以办理退款，但需要收款方也就是您这里向支付平台提出申请，平台第二天就可以退款给我。



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月2日(周日) 凌晨0:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
你一周后如果没消息再来问我. 我这边有消息了会第一时间通知您.

**Comment by David68326** [comment]
又过去快两周了，我的退款还没有收到，请解决。


蓝灯依然无法正常使用，请不要再拖了，谢谢！



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月2日(周日) 中午1:59
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
@David68326 您不客气. 尊重客户的体验感是我们的首要原则. 我们一定会善待我们的用户的. 谢谢您对蓝灯的支持!

**Comment by David68326** [comment]
退款为什么没动静了？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月2日(周日) 凌晨0:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
你一周后如果没消息再来问我. 我这边有消息了会第一时间通知您.

**Comment by David68326** [comment]
请问退款可以办理了吗？？？



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月2日(周日) 凌晨0:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
你一周后如果没消息再来问我. 我这边有消息了会第一时间通知您.

**Reply by MaxMKMax** [reply]
@David68326 您好. 您的情况我们在第三方系统无法退款. 但最好的方法是您从银行那里申请退款.
我可以再帮您问问看吧. 但不一定能. 我尽量尽力帮您.

**Comment by David68326** [comment]
我上次的邮件不是发第三方的回复的截图给你了吗？一年以内的订单都可以办理退款，只需要由你们收款方发起申请就可以。我的付款和你们的收款不是在同一个第三方平台上的吗？钱你们收走了，我让银行退钱给我？你们不把收到的钱退出来银行会拿他们的自己的钱退给我吗？你们做公司的这个道理都搞不明白吗？真不懂的话我告诉你们流程：你们在第三方平台收我钱的订单里发起退款申请并支付退款给他们，他们第二天就可以退款到我的银行卡上，就是这么简单。这样说能明白吗？都是做网络生意的，请不要再用这种没意义的理由推脱。


再说一遍，蓝灯到现在还是无法正常使用，请马上退款，谢谢！

---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年6月27日(周四) 凌晨0:07
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 请按承诺退款 (Discussion #536)




 
@David68326 您好. 您的情况我们在第三方系统无法退款. 但最好的方法是您从银行那里申请退款.
 我可以再帮您问问看吧. 但不一定能. 我尽量尽力帮您.
